x509/Makefile

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Why the Makefile here?  This won't get built unless someone specifically calls it, right?
My system does not have the binaries to generate the certs, so running the script directly does not
work either.  I think that is okay for now though, until we get serious about requiring TLS.